homeassistant/components/ecovacs/config_flow.py

```diff
@@ -46,14 +89,42 @@ async def _validate_input(
     try:
         await authenticator.authenticate()
     except ClientError:
-        _LOGGER.debug("Cannot connect", exc_info=True)
+        _LOGGER.exception("Cannot connect")
```

Comment:
We normally don't include the stack trace for known exceptions.

Reply:
Will revert it to include it only on debug